main.go

```diff
@@ -227,6 +227,28 @@ func handleRequest(defaultConfig *libhoney.Config, w http.ResponseWriter, req *h
 		}
 		return
 	}
+	// Load potential custom query parameters
+	// e.g. API key, dataset name, and honeycomb host
+	cfg := &libhoney.Config{}
+	query := req.URL.Query()
+	key := query.Get("key")
+	if key != "" {
+		cfg.APIKey = key
+	} else {
+		cfg.APIKey = defaultConfig.APIKey
+	}
+	dataset := query.Get("dataset")
+	if dataset != "" {
+		cfg.Dataset = dataset
+	} else {
+		cfg.Dataset = defaultConfig.Dataset
+	}
+	host := query.Get("host")
```

Comment:
Wouldn't this be better as `apiHost` or `api_host`

Reply:
Good idea. I was originally thinking that `host` or `key` was clear enough, but `api_host` and `api_key` match more with the environment variables, so are probably a better idea.